src/bin/client/cli.rs

```diff
@@ -81,6 +81,8 @@ pub(crate) fn build() -> clap::App<'static, 'static> {
                  "(optional) the timeout for this job in minutes. If the job doesn't \
                   complete within TIMEOUT minutes of entering the \"running\" state, \
                   the job killed.")
+                (@arg MAX_FAILURES: --max_failures +takes_value
```

Comment:
```suggestion
                (@arg MAX_FAILURES: --max_failures +takes_value requires[RETRY]
```

I don't think the new flag makes sense without retries...